addon/components/form.js

```diff
@@ -21,13 +21,14 @@ import validateView, {validateModel} from '../validator/index'
 import {deemberify, recursiveObjectCreate} from '../utils'
 
 const builtinRenderers = {
-  PropertyChooser: 'frost-bunsen-property-chooser'
+  'PropertyChooser': 'frost-bunsen-property-chooser',
+  'select': 'frost-bunsen-input-select',
+  'multi-select': 'frost-bunsen-input-multi-select'
```

Comment:
Yeah, probably change `PropertyChooser` to `property-chooser` it was always kinda weird that I made that `PascalCase` to begin with. I was pretty much referencing a React Component at the time, but since this is `ember-frost-bunsen` and not `react-frost-bunsen` and stuff will all be `dash-case` I think it makes sense to change it to `property-chooser`